### ISSUE TITLE
Replace ColorBrewer with d3-scale-chromatic?

### ISSUE THREAD

**Issue by martinRenou** [bqplot/bqscales#4]
It might be a good idea to use https://github.com/d3/d3-scale-chromatic instead of the ColorBrewer and ColorUtils scripts.

**Comment by SylvainCorlay** [comment]
Yes, I would love to discuss the transition away from Colorbrewer.
